rand_core/src/lib.rs

```diff
@@ -186,6 +186,19 @@ pub trait RngCore {
     /// 
     /// [`fill_bytes`]: trait.RngCore.html#method.fill_bytes
     fn try_fill_bytes(&mut self, dest: &mut [u8]) -> Result<(), Error>;
+
+    /// Number of bytes generated per round of this RNG.
+    ///
+    /// Some algorithms would benefit from knowing some basic properties about
+    /// the RNG. In terms of performance an algorithm may want to know whether
+    /// an RNG is best at generating `u32`s, or could provide `u64`s or more at
+    /// little to no extra cost.
+    ///
+    /// For many RNGs a simple definition is: the smallest number of bytes this
+    /// RNG can generate without throwing away part of the generated value.
+    ///
+    /// `bytes_per_round` has a default implementation that returns `4` (bytes).
+    fn bytes_per_round(&self) -> usize { 4 }
```

Comment:
Should we really give a default implementation here?

Did you forget to implement for Jitter?

Reply:
I was not sure. A disadvantage with a default implementation is that you can easily forget a wrapper.

> Did you forget to implement for Jitter?

`JitterRng::next_u32()` is about twice as fast as `next_u64()`, so 4 bytes would be the best fit there.